experiment/run_experiment.py

```diff
@@ -287,8 +287,10 @@ def filter_file(tar_info):
             return None
         return tar_info
 
-    experiment_filestore_path = os.path.join(config['experiment_filestore'],
-                                             config['experiment'])
+    os.environ['EXPERIMENT_FILESTORE'] = config['experiment_filestore']
```

Comment:
Leave a comment about why this is needed please.

Reply:
Done.